components/edit-collective/sections/SetupTwoFactorAuth.js

```diff
@@ -0,0 +1,337 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/react-hoc';
+import { Info } from '@styled-icons/feather/Info';
+import { Field, Form, Formik } from 'formik';
+import QRCode from 'qrcode.react';
+import { defineMessages, FormattedMessage, injectIntl } from 'react-intl';
+import speakeasy from 'speakeasy';
+import styled from 'styled-components';
+
+import { confettiFireworks } from '../../../lib/confettis';
+import { getErrorFromGraphqlException } from '../../../lib/errors';
+import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
+import { compose } from '../../../lib/utils';
+
+import Container from '../../Container';
+import { Box, Flex } from '../../Grid';
+import Loading from '../../Loading';
+import LoadingPlaceholder from '../../LoadingPlaceholder';
+import MessageBox from '../../MessageBox';
+import StyledButton from '../../StyledButton';
+import StyledInput from '../../StyledInput';
+import StyledInputField from '../../StyledInputField';
+import StyledTooltip from '../../StyledTooltip';
+import { H2, H3, P } from '../../Text';
+import { withUser } from '../../UserProvider';
+
+const messages = defineMessages({
+  errorWrongLength: {
+    id: 'TwoFactorAuth.Setup.Form.ErrorLength',
+    defaultMessage: 'Incorrect code length. Please re-enter your 6-digit code.',
+  },
+  errorWrongFormat: {
+    id: 'TwoFactorAuth.Setup.Form.ErrorFormat',
+    defaultMessage: 'Incorrect code format. Please enter only numbers.',
+  },
+  inputLabel: {
+    id: 'TwoFactorAuth.Setup.Form.InputLabel',
+    defaultMessage: 'Please enter your 6-digit code without any dashes.',
+  },
+});
+
+const content = () => (
+  <div>
+    <P fontSize="Caption" lineHeight="Caption">
+      <FormattedMessage
+        id="TwoFactorAuth.Setup.AppInfo"
+        defaultMessage="You can use apps such as Google Authenticator, Authy, 1Password, LastPass, or Microsoft Authenticator to scan the QR code and receive a code to input."
+      />
+    </P>
+  </div>
+);
+
+const TokenBox = styled(Box)`
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+`;
+
+class SetupTwoFactorAuth extends React.Component {
+  static propTypes = {
+    /** From intl */
+    intl: PropTypes.object.isRequired,
+    /** From graphql query */
+    addTwoFactorAuthTokenToIndividual: PropTypes.func.isRequired,
+    data: PropTypes.object,
+    /** From parent component */
+    slug: PropTypes.string,
+    collectiveName: PropTypes.string,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      tokenAdded: null,
+    };
+
+    this.submit = this.submit.bind(this);
+  }
+
+  componentDidMount() {
+    const options = {
+      name: `${this.props.collectiveName}'s Open Collective`,
```

Comment:
Where will this name appear?

Reply:
In the authenticator app
![IMG_8158](https://user-images.githubusercontent.com/37520401/87305682-cabfc180-c50e-11ea-9916-0c7f2a4f6fc7.jpg)